packages/replayio/src/utils/recordings/getRecordings.ts

```diff
@@ -274,11 +273,20 @@ export function getRecordings(processGroupIdFilter?: string): LocalRecording[] {
           return false;
         }
 
-        if (!recording.metadata.host) {
-          // Ignore new/empty tab recordings (see TT-1036)
-          // Note that we filter all "empty" recordings, not just root recordings,
-          // because Chrome loads its default new tab content via an <iframe>
-          return false;
+        switch (recording.recordingStatus) {
```

Comment:
nit: this looks a little bit verbose right now since this whole switch has only one case and its body has an additional condition. It would make sense to combine this in a single if statement (or even into `return condition`)

Reply:
Agreed it's verbose. When I write code that does something based on an enum, I lean towards `switch` even if it's only a single case because the pattern stands out more clearly to me (but I know this is just subjective).